client/components/login/login.test.js

```diff
@@ -64,10 +65,20 @@ describe("<Login /> rendering", () => {
 describe("<Login /> interactions", () => {
   let props;
   let wrapper;
+  let originalError;
+  let lastConsoleOutuput;
   beforeEach(() => {
+    originalError = console.error;
+    lastConsoleOutuput = null;
+    console.error = (data) => {
+      lastConsoleOutuput = data;
+    };
     props = createTestProps();
     wrapper = shallow(<Login {...props} />);
   });
+  afterEach(() => {
+    console.error = originalError;
+  });
```

Comment:
This code helps in `console.error`s handling (`login.js` generates them among with `toast.error` on error):
 - doesn't throw an error to console, so doesn't fail tests
 - adds `lastConsoleOutput` to check if the information about error actually exists during testing